src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './TodoList';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 10603;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState('all');
+  const [isHidden, setIsHidden] = useState(false);
+  const [query, setQuery] = useState('');
+  const [error, setError] = useState<null | string>(null);
```

Comment:
You can just go with empty string initially. If the string will be empty it will have the same effect as null in terms of being truthy/falsy
```suggestion
  const [error, setError] = useState('');
```